RecoLocalTracker/SiPixelDigiMorphing/python/SiPixelDigiMorphing_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+siPixelDigisMorphed = cms.EDProducer(
```

Comment:
please replace with a clone from `siPixelDigiMorphing_cfi` generated with the fillDescriptions